sdk/src/sharedTest/java/io/customer/sdk/CustomerIOClientTest.kt

```diff
@@ -291,6 +291,36 @@ class CustomerIOClientTest : BaseTest() {
         )
     }
 
+    // addCustomProfileAttributes
+
+    @Test
+    fun addCustomProfileAttributes_givenProfileIdentified_expectDoNotIdentifyProfile() {
+        val givenAttributes = mapOf(String.random to String.random)
+
+        customerIOClient.addCustomProfileAttributes(givenAttributes)
+
+        // do not identify profile
+        verifyNoInteractions(backgroundQueueMock)
```

Comment:
I do not like the code used here to test if `identify()` was called. This will be addressed with [a refactor splitting up the `CustomerIOClient` class](https://github.com/customerio/issues/issues/7160). These lines of code could then be much easier to understand and easier to maintain. I imagine the code will be changed to something similar to this:

```kotlin
verify(profileRepository, never()).identify(givenIdentifier)
```